test/test_nn.py

```diff
@@ -3831,7 +3831,11 @@ def test(encoder_input_shape, decoder_input_shape,
                  src_mask_len=None, tgt_mask_len=None, memory_mask_size=None,
                  src_key_padding_mask_size=None, tgt_key_padding_mask_size=None,
                  memory_key_padding_mask_size=None,
-                 raises=False):
+                 src_is_causal=False, tgt_is_causal=False,
+                 memory_is_causal=False, raises=True):
+
+            raises=True
```

Comment:
Looks like this was left here by accident

Reply:
Actually - this is a trial balloon on removing raises as an argument and just always make it True. ;).

Reply:
> Hm got it, in that case could we just remove it from the function signature and delete the argument at the other places where it is used

That's the plan - I just want to make sure all tests pass, before I hack up lots of call sites, and then have to redo them. Understood that this crutch can't be upstreamed as-is. (the only way to run all unit tests is github, since I'm working under fbcode... )

Reply:
gotcha! just flagging :)